src/explorer/McpServersTreeItem.ts

```diff
@@ -66,14 +66,16 @@ export class McpServersTreeItem extends AzExtParentTreeItem {
 
         const mcpServers = await apimService.listMcpServers();
 
+        const children: AzExtTreeItem[] = [
+            new McpPassthroughTreeItem(this, mcpServers || [], this.root),
```

Comment:
once mcpServers is null, both McpPassthrogh and McpTransform are exist?

Reply:
The `McpPassthrough` and `McpTransform` tree item provides the entry point to create MCP server, that's why we need to always show them here. There's an open item to review the experiences. As synced offline, will merge this PR first and create new PR to adjust the experience if needed.